### DISCUSSION TITLE
What kind of ☁️ is often mistaken for 🛸?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1140] [Galaxy Trivia] [answered]
**Bonus:** What my mom told me not to put in my ears, according to the song by The Serendipity Singers? (*answer as emoji*)

**Comment by peterbaccaro** [comment]
**Lenticular clouds** have been said to be mistaken for UFOs, because many of them have the shape of a "flying saucer", with a characteristic "lens" or smooth, saucer-like shape.

According to the song by The Serendipity Singers my mommy told me not to put **beans in my ears** (🫘➡️👂👂).  The song entitled "Beans in My Ears" is a song written by Len Chandler that was a hit single in 1964 when covered by The Serendipity Singers.

**Comment by tomerf** [comment, marked as answer]
Lenticular clouds are lens-shaped or saucer-shaped clouds.

Bonus: 👩🗣️🚫🫘➡👂

**Comment by thuongtruong1009** [comment]
The type of ☁️ (cloud) that is often mistaken for 🛸 (a flying saucer) is a lenticular cloud.

According to the song by The Serendipity Singers, what you should not put in your ears is 🍝 (spaghetti).

**Comment by cdprf** [comment]
lol i never heard lenticular clouds, thanks.  chat gpt giving best ansvers 📦
